<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phonominom 1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/16x16/earth_music.png</normaloff>:/16x16/earth_music.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="Phonon::VideoPlayer" name="videoPlayer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/16x16/folder_music.png</normaloff>:/16x16/folder_music.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_2">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/16x16/earth_music.png</normaloff>:/16x16/earth_music.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Phonon::SeekSlider" name="seekSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Phonon::VolumeSlider" name="volumeSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
